### PR TITLE
[Added][BEL-3940] Widget usage in link creation

### PR DESCRIPTION
**Pull request by dom-techwriter**
:question: What
---

As part of [3940](https://belvoteam.atlassian.net/browse/BEL-3940), we wanted to add an additional usage section where we highlight the use of the widget.

:speech_balloon: Why
---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,18 +68,18 @@ from belvo.client import Client
...
+client = Client("your-secret-key-id", "your-secret-key", "sandbox")
 
 # Register a link 
-link = client.Links.create(
+id = client.Links.create(
     institution="banamex",
     username="johndoe",
     password="supersecret",
     access_mode=AccessMode.SINGLE
 )
```
Comment (benitocas88) [inline README.md:80]: @dom-techwriter when you create a link via SDK you will receive this response (example response):
id = `{'id': '7ae62cf5-f170-4adb-b566-d5b8e6def430', 'institution': 'banamex', ...}` so, in this case you need do something like id['id']. Ok in short words, the change is not necessary L74 and L82

Comment (dom-techwriter) [reply README.md:80]: Ah! Of course, you're right! I'm a complete idiot when it comes to programming, I'm sorry @benitocas88 :(

**Comment by dom-techwriter** [comment]
@benitocas88 can you check now and tell me what you think? :)

File: README.md [hunk 2]
```diff
@@ -49,7 +79,7 @@ link = client.Links.create(
...
 
 # Get all accounts
-client.Accounts.create(link["id"])
+client.Accounts.create(link=link.id)
```
Comment (benitocas88) [inline README.md:82]:
```suggestion
client.Accounts.create(link=link["id"])
```

**Review by benitocas88 — APPROVED** [review]
👍🏼 on programming side, looks good to me

**Pull request merged**